testdriver/testdriver.go

```diff
@@ -53,7 +53,7 @@ var (
 func init() {
 	v := os.Getenv(testsetup.EnvErrLogMatchWait)
 	if v == "" {
-		v = "30s"
+		v = "25s"
```

Comment:
I think we should probably vary this via a workflow env var for now.

Reply:
Reverted and moved to the docker based tests